integration_tests/mockApis/govukOneLogin.ts

```diff
@@ -4,6 +4,8 @@ import { Response } from 'superagent'
 import { createPublicKey } from 'crypto'
 import { getMatchingRequests, stubFor } from './wiremock'
 
+let idToken: string
```

Comment:
Declare this here so it can be set via the sign-in stub and then used in the sign-out one.